# main_config.yaml

# General settings
system:
  logging_level: DEBUG
  max_timing_records: 100
  model_directory: /home/user/pel_ws/models/  # Common model storage location
  camera_topic: webcam/image_raw
  # service_to_test: /detr_resnet_50_panoptic/detect  # Service to test
  # task_type: SegmentationTask
  service_to_test: /yolos_tiny/detect
  task_type: StandardDetectionTask
  request_interval: 3
# Annotators
annotators:
  - detr_resnet_50
  - yolos_tiny
  - detr_resnet_50_panoptic
  # - fruit_detector
  # - segmentation_model

# ROS parameters
ros:
  node_name: orvis_annotators_node
  queue_size: 1

# Shared thresholds
confidence_threshold: 0.5
iou_threshold: 0.45

# Add any other system-wide shared settings here